src/main/java/com/online/store/dao/OrderDaoImpl.java

```diff
@@ -0,0 +1,7 @@
+package com.online.store.dao;
```

Comment:
```suggestion
package com.online.store.dao.impl;
```
same for all dao implementations

Reply:
done